tools/ttysnoop.py

```diff
@@ -41,6 +44,10 @@ def usage():
     help="don't clear the screen")
 parser.add_argument("device", default="-1",
     help="path to a tty device (eg, /dev/tty0) or pts number")
+parser.add_argument("-s", "--datasize", default="4096",
+    help="size of the transmitting buffer (default 4096)")
```

Comment:
Previous default is 256, now we increased to 4096, maybe in commit messages on why such an increase?

Reply:
ok, I kept the 256, with the buf fix you pointed out, I'm getting nice snoop output on ncurses apps as well